profiles/base.yaml

```diff
@@ -609,7 +609,36 @@ discard_scrolls:
 - Strange Arrow
 
 theurgy_whitelist:
+# Tithe 5 silver at the local almsbox.  Requres tithe: true
+- tithe
+# Study the wall in the Crossing temple.
+- study_wall
+# Meditate at the Crossing guild refectory
+- refectory
+# Carve a prayer bead.
+- carve_bead
+# Find and plant a sirese seed.
+- sirese_seed
+# Take a holy bath, you savage.
```

Comment:
:+1:

Reply:
What do you think about rewriting this to not require two lists but one? It should be written in a way that it goes through all of the things and only `.selects` the whitelist or `.rejects` the blacklist. This would lend us to having less settings.